### ISSUE TITLE
Talkback does not announce app dismissed when you dismiss it

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#263]
```
What steps will reproduce the problem?
1. have apps in the recent apps screen
2. swipe them away with the 2 finger gesture.
3. talkback does not say app dismissed

What is the expected output? What do you see instead?
talkback should say name of app dismissed

What version of the product are you using? On what operating system?
latest talkback android 4.2.2

Please provide any additional information below.
talkback used to do this but something changed and it no longer does it. I 
don't no how many versions it was ago that it spoke this information.

```

Original issue reported on code.google.com by `jmwalto...@gmail.com` on 14 May 2013 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
Out of scope for this project.  I've filed a separate android bug to track this.
```

Original comment by `caseybur...@google.com` on 29 May 2013 at 7:38
- Changed state: **Invalid**
